services/src/pro/contexts/postgres.rs

```diff
@@ -880,6 +893,7 @@ mod tests {
                             tokio_postgres::NoTls,
                             TestDefault::test_default(),
                             TestDefault::test_default(),
+                            get_config_element::<Quota>().unwrap(),
```

Comment:
this method is only used in tests, right? Is it a good idea to use the settings.toml for tests?

Reply:
that's what the Settings-test.toml is there for.